go.mod

```diff
@@ -2,7 +2,10 @@ module github.com/argoproj/argo/v2
 
 go 1.13
 
-replace github.com/argoproj/argo/v2 => ./
+replace (
+	github.com/argoproj/argo/v2 => ./
+	github.com/argoproj/pkg v0.3.0 => ../pkg
```

Comment:
Compilation errors will be due to this change, which I have made to test my `argoproj/pkg` changes locally